pages/login/style.css

```diff
@@ -0,0 +1,44 @@
+.ContentWrapper {
+	margin: 2rem 1rem 1rem 1rem;
+}
+.ContentWrapper > a {
+	display: block;
+	text-align: right;
+	text-decoration: none;
+	margin-bottom: 2.5rem;
+}
+
+.Link {
+	color: var(--neutral-neutral-700, #344054);
+}
+
+.Center {
+	text-align: center;
+}
+
+div > p {
+	color: var(--neutral-neutral-500, #667185);
+}
+
+h1 {
+	margin-bottom: 0.5rem;
+}
+
+form {
+	margin-top: 2.5rem;
+}
+
+.InputAndLabel {
```

Comment:
I don't think we need to override the properties of the components, just add the ones that we need, if you see inconsistencies with the Component and the Implementation let's take it to Design so we can keep the same styles.

Reply:
addressed in https://github.com/whamester/WMDD-4885-Project/pull/28/commits/48ba17dbf63f925cec4f9b39bd0c0935576f5b7e